### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by mecevit**
Updated documentation

### CODE REVIEW

File: examples/order_review_prediction/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Predicting Order Reviews in dbt DAG
+# Predicting Order Review Scores in dbt DAG
```
Comment (volkangurel) [inline examples/order_review_prediction/README.md:1]:
```suggestion
# Predicting Order Review Scores in the dbt DAG
```

File: README.md [hunk 2]
```diff
@@ -69,3 +68,83 @@ Check out the examples we have prepared for you:
...
+| --------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `MODEL_TYPE`    | Type of the model your want to train. There are two options: <br/> - `classifier`: A model to predict classes/labels or categories such as spam detection<br/>- `regressor`: A model to predict continious outcomes such as CLV prediction. |
+| `FEATURES`    | Input column names as a list to train your AutoML model.                                                                                                                                                                                    |
+| `TARGET`    | Target column that you want to predict.                                                                                                                                                                                                     |
+
+
+_Requirements:_
+- You need to put `layer_api_key` to your dbt profile to make AutoML work.
```
Comment (maasg) [inline README.md:95]:
```suggestion
- For AutoML to work, you need to add  the configuration `layer_api_key` with the API key of your Layer account into your dbt profile .
```

File: README.md [hunk 3]
```diff
@@ -69,3 +68,83 @@ Check out the examples we have prepared for you:
...
+
+_Example:_
+
+Check out [Order Review AutoML Project](https://github.com/layerai/dbt-layer/tree/mecevit/update-docs/examples/order_review_prediction):
```
Comment (maasg) [inline README.md:99]:
```suggestion
Check out [Order Review AutoML Project](https://github.com/layerai/dbt-layer/tree/main/examples/order_review_prediction):
```

File: examples/README.md [hunk 4]
```diff
@@ -4,9 +4,10 @@ These are example applications that use layer and dbt. Each one covers a differe
 
 ## Examples
 
-* [Titanic](./titanic) - Predicts the survivers of the Titanic disaster.
-* [Sentiment Analysis](./sentiment_analysis) - Conducts a multi-language sentiment analysis.
-* [Cloth Detector](./cloth_detector) - Detects cloths from product images using a pretrained computer vision model.
+- [Predicting survials of Titanic](https://github.com/layerai/examples-dbt/tree/main/titanic) - Predicts the survivals of the Titanic disaster.
```
Comment (maasg) [inline examples/README.md:7]:
```suggestion
- [Predicting survials of Titanic](https://github.com/layerai/examples-dbt/tree/main/titanic) - Predicts the survival chance for passengers of the Titanic disaster.
```

File: examples/README.md [hunk 5]
```diff
@@ -4,9 +4,10 @@ These are example applications that use layer and dbt. Each one covers a differe
...
+- [Sentiment analysis of product reviews](https://github.com/layerai/examples-dbt/tree/main/sentiment_analysis) - An example that shows how to make multi-language sentiment analysis.
+- [Object detection in product images](https://github.com/layerai/examples-dbt/tree/main/cloth_detector) - Detects cloths from product images using a pretrained computer vision model.
```
Comment (maasg) [inline examples/README.md:9]:
```suggestion
- [Object detection in product images](https://github.com/layerai/examples-dbt/tree/main/cloth_detector) - Detects clothes from product images using a pre-trained computer vision model.
```

File: examples/order_review_prediction/README.md (hunk 1 above)
Comment (maasg) [inline examples/order_review_prediction/README.md]:
```suggestion
# Predicting Order Review Scores from within the dbt DAG
```

File: examples/order_review_prediction/README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 
-In this e-commerce example walk through, we will train a machine learning model to predict order reviews in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
+In this e-commerce example walk through, we will train a machine learning model to predict order review scores in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
```
Comment (maasg) [inline examples/order_review_prediction/README.md:3]:
```suggestion
In this e-commerce example walk-through, we train a machine learning model to predict order review scores integrated within a dbt DAG, using  the open source [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
```

File: examples/order_review_prediction/README.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
 
-We will use `layer.automl()` functionality to train an ML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
+We will use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).
```
Comment (maasg) [inline examples/order_review_prediction/README.md:5]:
```suggestion
We use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
```

File: examples/order_review_prediction/README.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
+In this e-commerce example walk through, we will train a machine learning model to predict order review scores in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
 
-We will use `layer.automl()` functionality to train an ML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
+We will use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
 
-This is all the SQL we need to train our ML model within the dbt DAG. You just pass Layer the model type, input features and the target column we want to predict:
+This is all the SQL you need to train your ML model within the dbt DAG. You just pass Layer the model type, input features and the target column we want to predict:
```
Comment (maasg) [inline examples/order_review_prediction/README.md:7]: (changing person) If we want to use `you` here, we should change all previous steps from `we` to `you` as well.

**Review by volkangurel — APPROVED** [review]

File: examples/order_review_prediction/README.md [hunk 9]
```diff
@@ -22,9 +22,9 @@ FROM {{ ref('training_data') }}
 ```
 
 Once you run this SQL:
-1. Layer will try many different machine learning models with many parameters to find the best one for your data
-2. Once the training done, Layer will register the model to your Layer account
-3. Then, you will be able to fetch the model and use it inside dbt DAG. [Click here](./models/predictions.sql) to see an example in this project.
+1. Layer will try different machine learning algorithms with many parameters to find the best one for your data
```
Comment (maasg) [inline examples/order_review_prediction/README.md:25]:
```suggestion
1. Layer tries different machine learning algorithms with many parameters to find the best one for your data
```

File: examples/order_review_prediction/README.md [hunk 10]
```diff
@@ -22,9 +22,9 @@ FROM {{ ref('training_data') }}
...
+2. Once the training done, Layer will register the model to your Layer account. [Click here](https://app.layer.ai/layer/order_review_prediction/models/review_score_predictor) for a sample AutoML model registered on Layer.
```
Comment (maasg) [inline examples/order_review_prediction/README.md:26]:
```suggestion
2. Once the training done, Layer registers the model to your Layer account. [Click here](https://app.layer.ai/layer/order_review_prediction/models/review_score_predictor) for a sample AutoML model registered on Layer. 
```

File: examples/order_review_prediction/README.md [hunk 11]
```diff
@@ -22,9 +22,9 @@ FROM {{ ref('training_data') }}
 ```
 
 Once you run this SQL:
-1. Layer will try many different machine learning models with many parameters to find the best one for your data
-2. Once the training done, Layer will register the model to your Layer account
-3. Then, you will be able to fetch the model and use it inside dbt DAG. [Click here](./models/predictions.sql) to see an example in this project.
+1. Layer will try different machine learning algorithms with many parameters to find the best one for your data
+2. Once the training done, Layer will register the model to your Layer account. [Click here](https://app.layer.ai/layer/order_review_prediction/models/review_score_predictor) for a sample AutoML model registered on Layer. 
+3. Then, you will be able to fetch the model and use it for predictions inside dbt DAG with `layer.predict()`. [Click here](./models/predictions.sql) to see a prediction example in this project.
```
Comment (maasg) [inline examples/order_review_prediction/README.md:27]:
```suggestion
3. Then, you can fetch the model and use it for predictions from within dbt DAG with `layer.predict()`. [Click here](./models/predictions.sql) to see a prediction example in this project.
```

File: examples/order_review_prediction/README.md [hunk 12]
```diff
@@ -63,7 +63,7 @@ layer-profile:
 ```
 
 
-4. Now, we are ready to run our dbt DAG. To get the code, clone this repo, and go to the `titanic` example folder:
+4. Now, we are ready to run our dbt DAG. To get the code, clone this repo, and go to the `order_review_prediction` example folder:
```
Comment (maasg) [inline examples/order_review_prediction/README.md:66]: Change _you/we_ corresponding to the choices made above.

**Review by maasg — COMMENTED** [review]
I left several review comments for your consideration. 
One additional note: We tend to write in future form when describing examples ("we will..."). I changed that to present form in this review. 
From experience, it's better to use present form to keep the users "inside" the action.

**Comment by mecevit** [comment]
Thanks @maasg. I have replaced `we` with `you` in all docs. I will keep that in mind writing docs.

**Review by maasg — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**